lib/reek/cli/input.rb

```diff
@@ -44,6 +44,14 @@ def sources_from_argv
       def source_from_pipe
         [$stdin]
       end
+
+      def argv
+        raise NotImplementedError
+      end
+
+      def configuration
+        raise NotImplementedError
```

Comment:
Also not sure about those 2 methods here. I mean, it's a module already with instance methods, so it's clear that it is used as a mixin. Raising NotImplementedError here is something I find rather confusing than helpful. I'd rather remove those.